javascript/regular_expressions.json

```diff
@@ -183,7 +191,9 @@
       },
       "character_escape": {
         "__compat": {
-          "description": "Character escapes (<code>\\n</code>, <code>\\r</code>, etc.)",
+          "description": "Character escape: <code>\\n</code>, <code>\\u{...}</code>",
```

Comment:
cc @bsmth This one is tricky, because I've included `\u{...}`, which is ES6 and therefore must have newer support than others. Could you double-check support for this feature? (I suspect it's the same as `\p{...}`.) It should be added as a sub-feature.

Reply:
>Could you double-check support for this feature?

Sure, I will take a look 👍🏻

Reply:
I have the following:

__Firefox:__
* Added in 46 ([Bugzilla](https://bugzilla.mozilla.org/show_bug.cgi?id=1135377), [Release notes](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/46#javascript))